common/http.go

```diff
@@ -774,7 +785,12 @@ func addFromBody(response *http.Response, value *reflect.Value, field reflect.St
 	var iVal interface{}
 	switch encoding {
 	case "binary":
-		value.Set(reflect.ValueOf(response.Body))
+		byteArr, e := ioutil.ReadAll(response.Body)
```

Comment:
for string/binary in open api 2.0, it means a stream "octet-stream", a stream might not be able to fully loaded at this time, it might still streaming from server. So I am a bit concern here about load the stream to a buffer. (BTW, it works in kubeconfig case).

I am thinking to return the stream back to server without close it if it is a binary, thoughts?

Reply:
Hi @jasonyin , yes that was also my thinking at the beginning, and I just saw later that the implementation is also used for object storage calls for example. So yes I think not closing the stream in these cases is the correct way. Should I change the implementation like that?

Reply:
Is the method common.ClostBodyIfValid autogenerated, or the callsites or both?

Reply:
all common code are not autogen, just other files using it.

Reply:
The getObject call in the objectstorage client does not have that close body call. It looks like to me, that it somehow already solved in some part of the generator. Maybe the same solution should be applicable here as well?

Reply:
@bonifaido , I think you can use this change to unblock yourself for now. I am going to do a fix on generation part which will be release out on next Thursday.

Reply:
Great, thanks @jasonyin, than we will wait for the fix next Thursday, until then we will use this fix in our branch.

Reply:
cool, I am closing this PR now.